### ISSUE TITLE
aria-label support

### ISSUE THREAD

**Issue by makerj** [shuyu/angular-material-fileinput#12]
Is it possible to use the `aria-label` attribute with angular-material-fileinput?

following messages are shown by chrome:
`ARIA: Attribute " aria-label ", required for accessibility, is missing on node`
`<input type="file" accept="" ng-disabled="isDisabled" class="lf-ng-md-file-input-tag ng-scope md-input" onchange="angular.element(this).scope().onFileChanged(this)" id="input_8" aria-disabled="false">`

**Comment by shuyu** [comment]
ok I will figure out this problem !!
